python/ray/autoscaler/aws/tests/aws_config.yaml

```diff
@@ -2,7 +2,7 @@ base_image: {{ env["RAY_IMAGE_NIGHTLY_CPU"] | default("anyscale/ray:nightly-py37
 debian_packages: []
 
 python:
-  pip_packages: []
+  pip_packages: ["botocore>=1.29"]
```

Comment:
Does this mean `ray[default]` is broken right now? If so, can we change it there?

Reply:
No, I didn't understand the root cause and could never reproduce the error from scratch.  `ray[default]` in a blank conda environment is fine, and `ray[default]` on top of an Anyscale ray nightly image is also fine